R/03-geojob-obj.R

```diff
@@ -77,7 +77,7 @@ setMethod("geojob", signature("XMLDocument"), function(xml, ...) {
 #'@aliases geojob,geojob-method
 setMethod("geojob", signature("character"), function(xml, ...) {
   #parse based on xml class
-  if(startsWith(xml, "http")){
+  if(length(xml == 1) && any(startsWith(x = xml, prefix= c("http", "www")))){
```

Comment:
I don't know that this check is the best way to check for a url vs. xml.  I think it would probably be easier to look for the signs of xml (<> and such) and assume url otherwise.  If anything, I don't think www gains you much here (that subdomain is increasingly rare).